tests/storage/cdi_import/conftest.py

```
@@ -223,3 +227,16 @@ def dvs_and_vms_from_public_registry(unprivileged_client, namespace, storage_cla
             vm.clean_up()
         for dv in dvs:
             dv.clean_up()
+
+
+@pytest.fixture()
```

Ruff (0.15.10)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)